docs/Push-Server-Database.md

```diff
@@ -35,7 +35,7 @@ Stores push tokens specific for a given device.
 | `activation_id`             | VARCHAR(37)  | index                             | Activation ID associated with given push token record.                                                                                |
 | `user_id`                   | INTEGER      | index                             | Associated user ID.                                                                                                                   |
 | `app_id`                    | INTEGER      | index                             | Associated application ID.                                                                                                            |
-| `platform`                  | VARCHAR(30)  | -                                 | Mobile OS Platform ("ios", "android").                                                                                                |
+| `platform`                  | VARCHAR(30)  | -                                 | Mobile OS Platform ("apns", "fcm", "hms").                                                                                             |
```

Comment:
Strictly speaking we still supporting the `ios` and `android` values - for old devices